server/src/test/java/org/opensearch/index/translog/RemoteFSTranslogTests.java

```diff
@@ -607,10 +607,6 @@ public void testMetadataFileDeletion() throws Exception {
         long oldPrimaryTerm = primaryTerm.get();
         long newPrimaryTerm = primaryTerm.incrementAndGet();
 
-        // Check all metadata files corresponds to old primary term
```

Comment:
Why have we removed this check on primary term names at 2 places in this file?

Reply:
It doesn't contain primary term name now .  Hence , it is no longer needed .